src/test/scala/com/microsoft/ml/spark/cognitive/split1/TextAnalyticsSDKSuite.scala

```diff
@@ -103,6 +126,17 @@ class TextSentimentSuiteV4 extends TestBase with DataFrameEquality with TextKey
       data(0).get(2).toString == "negative")
     assert(data(1).get(0).toString == "positive")
   }
+
+  test("Sentiment Analysis - Assert Confidence Score") {
+    val replies = detector.transform(batchedDF)
+      .select("output.result.sentences", "output.result.confidenceScores")
+      .collect()
+    val confidence = replies.map(row => row.getList(1))
+    assert(confidence(0).get(0).toString == "[1.0,0.0,0.0]")

```

Comment:
As we discussed, could we update this to check the confidence score for negative/positive/neural  --> if the input sentence is positive, just check whether the positive score is larger than 0.5